pages/blog/microcks-asyncapi-part2.md

```diff
@@ -0,0 +1,100 @@
+
+Mocking event-driven architecture using Microcks is a game changer as you do not need to write code nor set up complex infrastructure! Your consumers are able to receive messages in the minute and testing some changes is just one commit away: you update the AsyncAPI specification in the Git repository and Microcks will take care of updating everything! It’s even capable of providing and managing the Kafka infrastructure thanks to the excellent [Strimzi.io](https://strimzi.io/) operator if you wish!
+
+**Our second feature is testing or how to make your delivery lifecycle reliable.**
+
+Obviously, as the number of event producers and subscribers is exploding: managing changes and taking care of versioning compatibility is essential. And what about checking that business rules implying event triggering are correctly implemented ? The fact it produces syntactically correct events and all this in a fully automated way based on each change and new commit in your source code repository ?
+
+Again this is all provided by Microcks thanks to its capability to interoperate with your CI/CD pipeline using our plugins for [Jenkins](https://microcks.io/documentation/automating/jenkins/), [Tekton](https://microcks.io/documentation/automating/tekton/) or [any other CI pipeline technology like GitLab](https://microcks.io/documentation/automating/cli/). You’ll typically use these plugins to trigger a Kafka test in Microcks.
+
+![microcks-kafka-testing](/img/posts/microcks-asyncapi-part2/microcks-kafka-testing.webp)
+
+In Microcks, testing Kafka endpoints means connecting to a remote Kafka topic on an existing broker in the organisation, listening for incoming messages and checking that received messages are valid regarding the event-based API schema that is referenced in your source of truth: the AsyncAPI specification. You can find further technical details on the blog post [mocking and testing Apache Kafka API using Microcks](https://microcks.io/blog/apache-kafka-mocking-testing/).
+
+Testing event-driven architecture is no longer a nightmare with Microcks! Microcks is able to connect to the Kafka brokers in your organization and tell you if received messages are valid according to your specification. No drifting risks anymore or way to introduce regression in production! You’ll drive and control everything from your pipeline.
+
+**What are the business use-cases of AsyncAPI? Where can you use Microcks as an essential part of your toolchain?**
+
+As said before event-driven and asynchronous APIs are becoming mainstream because we truly understand now the level of decoupling - and thus power and agility - it brings within our products.  We see the need for asynchronous APIs - and the presence of Apache Kafka as the de facto standard for message brokering - everywhere :
```

Comment:
```suggestion
As said before, event-driven and asynchronous APIs are becoming mainstream because we truly understand the decoupling level - and thus power and agility - it brings within our products. We see the need for asynchronous APIs and Apache Kafka's presence as the de facto standard for message brokering - everywhere.
```